src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -80,6 +92,7 @@ class AWSResources:
     S3_ACTIONS: typing.ClassVar[set[str]] = {"s3:PutObject", "s3:GetObject", "s3:DeleteObject", "s3:PutObjectAcl"}
     S3_READONLY: typing.ClassVar[str] = "s3:GetObject"
     S3_REGEX: typing.ClassVar[str] = r"arn:aws:s3:::([a-zA-Z0-9\/+=,.@_-]*)\/\*$"
+    S3_BUCKET: typing.ClassVar[str] = r"((s3:\/\/|s3a:\/\/)([a-zA-Z0-9+=,.@_-]*))\/.*$"
```

Comment:
can we modify `S3_PATH_REGEX` instead

Reply:
the other place that `S3_PATH_REGEX` is used is in `aws/access.py`, and that only checks that it matches an s3 path

Reply:
Slightly different uses. S3Path extracts the whole path. Bucket extracts just the bucket path.